build/docker/publish/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM gcc:8 as build
+
+RUN apt update && apt upgrade -y && apt install flex bison cmake -y
+
+WORKDIR /usr/src/thrift
+
+COPY . /usr/src/thrift
+
+RUN mkdir cmake-build && cd cmake-build && cmake -DWITH_SHARED_LIB=off .. && make
```

Comment:
Is there a way to build, drop the build directory, and squash the layers so you don't have to ship all the intermediary bits that make the container larger?

Reply:
it's already done there output image is based on ubuntu image and contains only thrift binary
`FROM ubuntu 
COPY --from=build /usr/src/thrift/cmake-build/bin/thrift /bin/thrift
WORKDIR /thrift
ENTRYPOINT ["thrift"]`